src/public.rs

```diff
@@ -4,12 +4,56 @@ use std::{thread::sleep, time::Duration};
 use strum::IntoEnumIterator;
 use strum_macros::EnumIter;
 
+#[cfg(feature = "serde")]
+use lazy_static::lazy_static;
+#[cfg(feature = "serde")]
+use regex::Regex;
+#[cfg(feature = "serde")]
+use serde::{
+    de::{Deserializer, Error},
+    Deserialize, Serialize,
+};
+#[cfg(feature = "serde")]
+use thiserror::Error;
+
 pub enum BlockInput {
     Block,
     DontBlock,
 }
 
+#[cfg(feature = "serde")]
+lazy_static! {
+    static ref OTHER_KEY: Regex = Regex::new(r"OtherKey(\d+)").unwrap();
```

Comment:
Is it possible to use `once_cell` here instead of `lazy_static` to avoid the extra dependency?

Reply:
Done